plugins/mac-syphon/CMakeLists.txt

```diff
@@ -67,7 +67,7 @@ target_compile_options(
   mac-syphon
   PRIVATE -include
           ${CMAKE_CURRENT_SOURCE_DIR}/syphon-framework/Syphon_Prefix.pch
-          -fobjc-arc)
+          -fobjc-arc -Wno-deprecated-declarations)
```

Comment:
I don't think we should just ignore all deprecations here but rather fix them or ignore them individually.
I addressed the one I got in #6767, do you have any others?

Reply:
I don't, CI will tell.